lib/matplotlib/tests/test_patches.py

```diff
@@ -396,6 +396,18 @@ def test_connection_patch():
     ax2.add_artist(con)
 
 
+def test_connection_patch_fig():
```

Comment:
Does this need some sort of image test as well?

Reply:
Previously it would error out, so I added this just as a smoke test to make sure it doesn't. The functionality should be sufficiently tested by the other image test.  
I mean, I could of course add this to the existing image test (just being lazy here); if people find it necessary, let me know.

Reply:
Up to you if you think this will keep it from getting broken is the only reason we have tests

Reply:
Yes, I'd say this is sufficient to test what's being changed here. Of course one could argue that in general we do not test all of the possible combinations of parameters, and for each with and without contrained_layout, etc...